python/nano/tutorial/notebook/training/pytorch-lightning/accelerate_pytorch_lightning_training_multi_instance.ipynb

```diff
@@ -0,0 +1,260 @@
+    "    val_transform = transforms.Compose([transforms.Resize(256),\n",
+    "                                        transforms.CenterCrop(224),\n",
+    "                                        transforms.ToTensor(),\n",
+    "                                        transforms.Normalize([0.485, 0.456, 0.406],\n",
+    "                                                             [0.229, 0.224, 0.225])])\n",
+    "\n",
+    "    # apply data augmentation to the tarin_dataset\n",
+    "    train_dataset = OxfordIIITPet(root=\"/tmp/data\", transform=train_transform, download=True)\n",
+    "    val_dataset = OxfordIIITPet(root=\"/tmp/data\", transform=val_transform)\n",
+    "\n",
+    "    # obtain training indices that will be used for validation\n",
+    "    indices = torch.randperm(len(train_dataset))\n",
+    "    val_size = len(train_dataset) // 4\n",
+    "    train_dataset = torch.utils.data.Subset(train_dataset, indices[:-val_size])\n",
+    "    val_dataset = torch.utils.data.Subset(val_dataset, indices[-val_size:])\n",
+    "\n",
+    "    # prepare data loaders\n",
+    "    train_dataloader = DataLoader(train_dataset, batch_size=32)\n",
+    "    val_dataloader = DataLoader(val_dataset, batch_size=32)\n",
+    "\n",
+    "    return train_dataloader, val_dataloader"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model = MyLightningModule()\n",
+    "train_loader, val_loader = create_dataloaders()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; _The definition of_ `MyLightningModule` _and_ `create_dataloaders` _can be found in the_ [runnable example](https://github.com/intel-analytics/BigDL/tree/main/python/nano/tutorial/notebook/training/pytorch-lightning/accelerate_pytorch_lightning_training_multi_instance.ipynb)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To enable multi-instance training, you could simply **import BigDL-Nano Trainer, and set** `num_processes` **to an integer larger than 1**:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from bigdl.nano.pytorch import Trainer\n",
+    "\n",
+    "trainer = Trainer(max_epochs=5, num_processes=2)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "> 📝 **Note**\n",
+    ">\n",
+    "> By setting `num_processes`, Nano will launch the specific number of processes to perform data-parallel training. By default, CPU cores will be automatically and evenly distributed among processes to avoid conflicts and maximize training throughput. If you would like to specifiy the CPU cores used by each process, You could set `cpu_for_each_process` to a length `num_processes` list, in which each item is a list of CPU indices.\n",
```

Comment:
"You could set cpu_for_each_process to a length num_processes list" is a little strange.

Reply:
Have changed to:
set `cpu_for_each_process` to a list of `num_processes` length